src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
```

Comment:
Add free space between siblings. Repeat, please with blocks below 

```suggestion
    </div>
    
    <div class="stars stars--1">
```

Reply:
@GUSILLUS 
You have to add new line between `</div>` and `<div class="stars stars--1">` but not before first child inside block :)